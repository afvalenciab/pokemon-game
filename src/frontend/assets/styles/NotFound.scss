.notfound {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  height: 100vh;
  background: rgb(223, 176, 8);
}