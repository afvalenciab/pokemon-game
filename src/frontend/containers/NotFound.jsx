import React from 'react';
import snorlax from '../assets/static/Pokemon_snorlax.svg';
import '../assets/styles/NotFound.scss';

const NotFound = () => {
  return(
    <section className="notfound">
      <h1>Not Found</h1>
      <figure>
        <img src={snorlax} alt="snorlax"/>
      </figure>
    </section>
  );
}

export default NotFound;
