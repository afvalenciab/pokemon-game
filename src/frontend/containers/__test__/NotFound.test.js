import React from 'react';
import { create } from 'react-test-renderer';
import NotFound from '../NotFound';

describe('NotFound testing,', () => {
  test('Match Snapshot', () => {
    const notFound = create(<NotFound />);
    expect(notFound.toJSON()).toMatchSnapshot();
  });
});