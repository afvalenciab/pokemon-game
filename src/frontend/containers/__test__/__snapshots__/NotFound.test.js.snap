// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound testing, Match Snapshot 1`] = `
<section
  className="notfound"
>
  <h1>
    Not Found
  </h1>
  <figure>
    <img
      alt="snorlax"
      src="test-file-stubs"
    />
  </figure>
</section>
`;
